doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -1877,3 +1883,5 @@ \subsubsection{Inputs}\label{inputs-17}
     Lshaped Zone:South Wall,Lshaped Zone:East Wall,0.101310,
   <snip>
 \end{lstlisting}
+\]
```

Comment:
Similar to [above](https://github.com/NREL/EnergyPlus/pull/5762/files#r71266857)...